webclient/xml_generator/generator.py

```diff
@@ -504,6 +503,6 @@ def generate_document(self):
     def __init__(self, document_object):
         self.document_object = document_object
         ET.register_namespace("xsi", "http://www.w3.org/2001/XMLSchema-instance")
-        ET.register_namespace("gml", "https://www.opengis.net/gml/3.2")
+        ET.register_namespace("gml", "http://www.opengis.net/gml/3.2")
```

Comment:
V čem je rozdíl?

Reply:
Problém je ve třech věcech:
- podle definice je správně pouze http://www.opengis.net/gml/3.2 - to říká standard GML
- v datech je https://www.opengis.net/gml/3.2 v xmlns atributu a http://www.opengis.net/gml/3.2 v lokátoru (ten tam ale být nemusí, tak jsem ho odstranil)
- v XSD v typu je správně http://www.opengis.net/gml/3.2, takže to nyní nevaliduje - varianta by byla změnit pouze XSD; pak by fungovala validace, ale to by zase nebylo podle standardu

Bohužel jsem si toho měl všimnou dříve. Asi se Vám to tam někdy zamotalo při tvorbě generátoru a já to nikdy pořádně nedotestoval.

Reply:
@pesikj Prosím o případné potvrzení, ať to můžeme mergenout, pokud tam nevidíte nějaký problém.